src/libmongoc/doc/mongoc_collection_command_simple.rst

```diff
@@ -48,35 +48,9 @@ This function does not check the server response for a write concern error or wr
 Example
 -------
 
-The following is an example of executing the collection stats command.
-
-.. code-block:: c
-
-  #include <bson/bson.h>
-  #include <mongoc/mongoc.h>
-  #include <stdio.h>
-
-  static void
-  print_collection_stats (mongoc_collection_t *collection)
-  {
-     bson_error_t error;
-     const char *name;
-     bson_t *cmd;
-     bson_t reply;
-
-     name = mongoc_collection_get_name (collection);
-     cmd = BCON_NEW ("collStats", BCON_UTF8 (name));
-
-     if (mongoc_collection_command_simple (
-            collection, cmd, NULL, &reply, &error)) {
-        str = bson_as_canonical_extended_json (&reply, NULL);
-        printf ("%s\n", str);
-        bson_free (str);
-     } else {
-        fprintf (stderr, "%s\n", error.message);
-     }
-
-     bson_destroy (&reply);
-     bson_destroy (cmd);
-  }
+The following is an example of executing the ``ping`` command.
 
+.. literalinclude:: ../examples/example-collection-command.c
+   :caption: Example use of mongoc_collection_command
```

Comment:
```suggestion
```
Caption is redundant given the "The following is an example..." note above. Suggest picking one or the other.